pmm-app/src/pmm-update-panel/module.ts

```diff
@@ -77,13 +77,14 @@ export class PanelCtrl extends MetricsPanelCtrl {
         const body = document.querySelector('body');
         const timeDiff = Date.now() - Number(localStorage.getItem('lastCheck'));
         const escKeyCode = 'Escape';
+        const isCurrentMoreThanNext = +$scope.currentVersion.split('.').join("") > +$scope.nextVersion.split('.').join("");
```

Comment:
This is very naïve comparison: 2.0.0 < 1.15.1.
Better compare major1 <= major2 &&  minor1 <= minor2 && bugfix1 < bugfix2;